### PR TITLE
Front-39

### PR DESCRIPTION
**Pull request by SimonjShurety**
I used a quote generating API to simulate the backend data. The animation styling still needs some fine tuning and the page flipping book itself isn't exactly as I hoped but we can always switch in another animation.

### CODE REVIEW

**Review by koral14 — APPROVED** [review]
Great job! The flipbook looks very nice!!!

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]

**Comment by Ekaterina-Bondareva** [comment]
Good job Simon! I suppose you need to remove display: flex from body because it breaks display (big screens) page styling.  

Also I found out that we have a problem with Header is not render correctly. I suppose the problem could be that it was fixed after you start working your ticket and you do not need make any changes.

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]
Nice work Simon! 
1. Could you please merge latest changes from main to your brunch so we will how everything looks like with all other updates? 
2. According to the ticket `we should prevent interaction with the page while loading is set to true. ` but I still can interact with page during loading, so it is not work.

File: src/App.js [hunk 1]
```diff
@@ -116,6 +131,26 @@ const App = () => {
...
+                        <button
+                            className="btn get-quote"
+                            onClick={getRandomQuote}
+                        >
+                            Loading Spinner Quote Button (click here)
+                        </button>
+                    </div>
+                    {loading ? (
+                        <LoadingSpinner />
+                    ) : (
```
Comment (Ekaterina-Bondareva) [inline src/App.js:145]: This behaviour should probably be hidden inside` LoadingSpinner `component with `loading` and `setLoading` passed as prop if I understood ticket correctly.

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]
Nice work Simon!

Could you please merge latest changes from main to your brunch so we will how everything looks like with all other updates?
According to the ticket we should prevent interaction with the page while loading is set to true.  but I still can interact with page during loading, so it is not work.

File: src/App.js (hunk 1 above)
Comment (SimonjShurety) [reply src/App.js:136]: Remove ability to click on page while loading.

**Pull request closed without merging**